openstack/neutron/templates/etc/_neutron-policy.json.tpl

```diff
@@ -7,13 +7,13 @@
     "admin": "role:network_admin and rule:owner",
     "context_is_network_admin": "rule:context_is_admin or rule:admin",
     "context_is_editor": "rule:context_is_network_admin or rule:member",
-    "context_is_viewer":  "rule:context_is_editor or rule:viewer",
+    "context_is_viewer":  "rule:context_is_editor or rule:viewer or rule:context_is_securitygroup_admin or rule:context_is_securitygroup_viewer",
     "network_view_all": "role:network_viewer or role:member or role:network_admin or rule:context_is_admin",
 
     "network_owner": "tenant_id:%(network:tenant_id)s",
     "network_owner_or_owner": "rule:network_owner or rule:owner",
     "network_member": "role:member and rule:network_owner_or_owner",
-    "network_viewer": "role:network_viewer and rule:network_owner_or_owner",
+    "network_viewer": "(role:network_viewer and rule:network_owner_or_owner) or rule:context_is_securitygroup_viewer or rule:context_is_securitygroup_admin",
```

Comment:
`or rule:context_is_securitygroup_admin` is not necessary because `rule:context_is_securitygroup_viewer` contains that. Same above.

Reply:
Is admin is removed.